### PR TITLE
CIV-9127 Trial Arrangements - Has anything changed to the support

### PR DESCRIPTION
**Pull request by hmcts-version1-ignacio**
### The scope of this ticket is to answer the questions if anything has changed to the support or adjustment the defendant LiP would like to be considered. ###

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by maciejmodzelewski — APPROVED** [review]

File: src/main/views/features/dashboard/item-content.njk [hunk 1]
```diff
@@ -20,13 +20,11 @@
...
   {% if content.type === ClaimSummaryType.TITLE %}
     {% set object = '' %}
-    {% if content.data.classes %}
-      <h3 class="govuk-heading-m {{ content.data.classes }}">{{ t(content.data.text) }}</h3>
-    {% elif  content.data.variables %}
-       {% set object = content.data.variables %}
-       <h3 class="govuk-heading-m">{{ t(content.data.text,object) }}</h3>
+    {% if  content.data.variables %}
+      {% set object = content.data.variables %}
+      <h3 class="govuk-heading-m {{ content.data.classes }}">{{ t(content.data.text,object) }}</h3>
```
Comment (kurt-uk) [inline src/main/views/features/dashboard/item-content.njk:25]: do we no longer need this check for classes in? what happens if no classes are passed in?

File: src/main/services/features/caseProgression/trialArrangements/hasAnythingChanged.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import {Claim} from 'models/claim';
+import {UploadYourDocumentsSectionBuilder} from 'models/caseProgression/uploadYourDocumentsSectionBuilder';
+import {caseNumberPrettify} from 'common/utils/stringUtils';
+import {CASE_DOCUMENT_DOWNLOAD_URL} from 'routes/urls';
+import {DocumentType} from 'models/document/documentType';
+import {getSystemGeneratedCaseDocumentIdByType} from 'models/document/systemGeneratedCaseDocuments';
+
+export const getHasAnythingChanged = (claimId: string, claim: Claim) => {
+  const documentId = getDocumentId(claim, DocumentType.DEFENDANT_DEFENCE);
+  return new UploadYourDocumentsSectionBuilder()
```
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply src/main/services/features/caseProgression/trialArrangements/hasAnythingChanged.ts]: As they are common components, moved them to the utils' pageSectionBuilder and used that one.

File: src/main/views/features/dashboard/item-content.njk (hunk 1 above)
Comment (hmcts-version1-ignacio) [reply src/main/views/features/dashboard/item-content.njk:25]: Nothing happens if no classes are added. It is optional and we've been using it like this for the other components.

Comment (kurt-uk) [reply src/main/views/features/dashboard/item-content.njk:25]: 👍🏼

**Review by kurt-uk — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Pull request merged**